packages/utils/styling/src/getMarginAdjustment.win32.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Adjusts the margin of text so that it's centered within the layout area of the text.
+ * Due to how Segoe UI lays out, the text does not appeared centered within the text
+ * layout area without some fuding of margins.
+ * @returns Adjustment necessary for text
+ */
+export function getTextMarginTopAdjustment(): number {
+  return -1;
+}
+
+/**
+ * Adjusts the margin of text so that it's centered within the layout area of the text.
+ * Due to how Segoe UI lays out, the text does not appeared centered within the text
+ * layout area without some fuding of margins.
+ * @returns Adjustment necessary for text
+ */
+export function getTextMarginBottomAdjustment(): number {
+  return 1;
+}
+
+/**
+ * Adjusts the margin of text so that it's centered within the layout area of the text.
+ * Due to how Segoe UI lays out, the text does not appeared centered within the text
+ * layout area without some fuding of margins.
+ * @returns Adjustment necessary for text
+ */
+export function getTextMarginLeftAdjustment(): number {
+  return 1;
+}
+
+/**
+ * Adjusts the margin of text so that it's centered within the layout area of the text.
+ * Due to how Segoe UI lays out, the text does not appeared centered within the text
+ * layout area without some fuding of margins.
+ * @returns Adjustment necessary for text
+ */
+export function getTextMarginRightAdjustment(): number {
+  return -1;
+}
```

Comment:
I think we could combine all 4 into one function, not sure when you would need just one of them

Reply:
That's true. And if someone needed to override one field they could use spread operator or something. I'll do that.